### DISCUSSION TITLE
Using Philips Hue Bridge hardware

### DISCUSSION THREAD

**Discussion by DanCInOz** [Koenkk/zigbee2mqtt#13636] [Devices]
Has anyone tried to install zigbee2mqtt on a rooted Hue Bridge?
I've been struggling to get my Sonoff ZBDongle-E (EFR32MG21) working under z2m and the thought occurred to me that the instead of running z2m on my HA rPi, it might be possible to run it on a rooted Hue Bridge...

I've done some googling but can't seem to find an example of someone doing this...

**Comment by Koenkk** [comment]
This is not possible, I suggest to use one of the recommended dongles for the best experience: https://www.zigbee2mqtt.io/guide/adapters/#recommended

**Reply by DanCInOz** [reply]
Why not?     Is it a case of z2m not being able to talk to the Hue zigbee radio?

**Reply by Koenkk** [reply]
>  Is it a case of z2m not being able to talk to the Hue zigbee radio?

Exactly

**Reply by DanCInOz** [reply]
I had a quick look today...   It seems that the radio is accessible via a TTY device in the OpenWRT OS.    I guess I figured there should be a way.
